tjreports/site/views/reports/tmpl/default.php

```diff
@@ -431,7 +433,12 @@
 													}
 													else
 													{
-														echo '<td class="' . $key . '">'. $item[$key] .'</td>';
+														$isSendEmailClass = '';
```

Comment:
Change this to 
`$isSendEmailClass = (in_array($key, $this->emailColumn)) ? $emailColmClass : '';`

and
`echo "<td class=\"{$key} {$isSendEmailClass}  \">{$item[$key]}</td>";`